tests/testing_helpers.py

```diff
@@ -433,6 +438,8 @@ def build_mocked_server(name=None, config=None, global_conf=None, main_conf=None
     server.postgres.xlog_segment_size = DEFAULT_XLOG_SEG_SIZE
     server.path = "/test/bin"
     server.systemid = "6721602258895701769"
+    # Add server_version to comply with major PG server version contraint for incremental backups
```

Comment:
```suggestion
```

I'd leave this comment out, as that's something too specific for a test "fixture".